events/events.py

```diff
@@ -138,6 +138,11 @@ def generate_ical_file(generator):
 
         ical.add_component(ie)
 
+    # If the directory for the ICS file don't exist, make it
```

Comment:
Suggested fixes for comment: `# If the directory for the ICS file doesn't exist, create it`